### PR TITLE
Update upload files page

### PR DESCRIPTION
**Pull request by Malinfield**

### CODE REVIEW

File: src/pages/filesSystemPage/index.jsx [hunk 1]
```diff
@@ -175,15 +183,22 @@ export const FilesSystemPage = () => {
   }, [user]);
 
   const onBackButtonClick = () => {
-    navigate('/file-upload');
+    navigate(-1);
     dispatch(clearSearchAutocomplete());
     setSearchValue('');
   };
 
+  if (Object.keys(types).length === 0 || !human || !user) {
```
Comment (fifa-master) [inline src/pages/filesSystemPage/index.jsx:191]: usememo

File: src/pages/filesSystemPage/index.jsx [hunk 2]
```diff
@@ -175,15 +183,22 @@ export const FilesSystemPage = () => {
   }, [user]);
 
   const onBackButtonClick = () => {
-    navigate('/file-upload');
+    navigate(-1);
```
Comment (fifa-master) [inline src/pages/filesSystemPage/index.jsx:186]: prev variant better

File: src/pages/filesSystemPage/index.jsx [hunk 3]
```diff
@@ -33,13 +34,14 @@ import { ReactComponent as ListIcon } from '../../assets/list_view.svg';
...
 import { ReactComponent as SearchIcon } from './assets/search.svg';
-import { ReactComponent as SquareIcon } from './assets/square.svg';
+import { ReactComponent as BackIcon } from './assets/close.svg';
 
 import style from './style.module.scss';
 
 const MAX_FILE_SIZE = 268435456;
 
 export const FilesSystemPage = () => {
+  const [types, setTypes] = useState({});
```
Comment (fifa-master) [inline src/pages/filesSystemPage/index.jsx:44]: undefined better than empty object

File: src/pages/filesSystemPage/index.jsx (hunk 1 above)
Comment (Malinfield) [reply src/pages/filesSystemPage/index.jsx:191]: Why?

File: src/pages/filesSystemPage/index.jsx (hunk 2 above)
Comment (Malinfield) [reply src/pages/filesSystemPage/index.jsx:186]: No, we do not need redirect to '/file-upload' anymore

File: src/pages/filesSystemPage/index.jsx (hunk 1 above)
Comment (Malinfield) [reply src/pages/filesSystemPage/index.jsx:191]: where do u see calculation?

Comment (fifa-master) [reply src/pages/filesSystemPage/index.jsx:191]: Object.keys

Comment (Malinfield) [reply src/pages/filesSystemPage/index.jsx:191]: this is a small object and there are no frequent rerenders. i don't think it's necessary to put a usememo everywhere

Comment (Malinfield) [reply src/pages/filesSystemPage/index.jsx:191]: і can put this in a separate variable for better readability

Comment (fifa-master) [reply src/pages/filesSystemPage/index.jsx:191]: even if it is a small object, it will still be calculated equally for each render

**Review by fifa-master — APPROVED** [review]

**Pull request merged**